### ISSUE TITLE
Goodbye Atom. Hello VS Code.

### ISSUE THREAD

**Issue by FHU-yezi** [atom/atom#25505]
VS Code has more powerful features and extensions, let's move on.

**Comment by aalbacetef** [comment]
I don't contribute or anything to this project, but these posts are just ridiculous.

Can someone close this issue please?

**Comment by Zagur** [comment]
VS Spy Code. 
Better VSCodium https://vscodium.com/

Anyway, it's a shame to kill such a great project like Atom :(

**Comment by iamunadike** [comment]
Honestly Its a shame  Atom beats VScode 100% till 2morrow.  I have both vscode and atom on my localmachine but hardly like HArDLY do I use vscode. Its too bloated, consumes CPU , useless marketplace,etc
Atom is not as big but it hits the CORE developer need/
Bill and his evil of buying github and atom which was Defacto shows how evil that man is. He likes to Upset people to accept his useless products. Atom also works better on Linux and Vscode works better on windows. 
I dont LIKE vscode
I want atom and will COntinue to USE ATOM
Please Atom Community, Fork Atom and acquire all the resources,
The Atom project is too good to be Neglected , Microsoft wants to enforce Vscode and We all SAY NO
We all have choices and Editor Preferences , So WHY force anyone...

**Comment by kaosine** [comment]
I mean other choices exist and I'd recommend vscodium if you want a version without all the bloat. It'd be nice if they'd let the community take over atom since atom just feels a bit different than vs....its just not the same.

**Comment by iamunadike** [comment]
Your point is very candid and commendable,. Atom since github was abandoned by microsoft siince they needed to market vscode,
It is the community of developer out of free time and effort that contribute
Now atom started coming to life again , gaining popularity especially with 1.60 release,
they noticed this Since VSCode TRACKS YOU BRO and they felt well... Its about time, They should force everyone to use useless VScode . .... so you get it now

**Comment by StaggerLeee** [comment]
> I mean other choices exist and I'd recommend vscodium if you want a version without all the bloat. It'd be nice if they'd let the community take over atom since atom just feels a bit different than vs....its just not the same.

Why not just to disable it in settings ?
[https://code.visualstudio.com/docs/getstarted/telemetry#_disable-telemetry-reporting](url)

**Comment by AndrusGerman** [comment]
I am sincerely proud of everyone who worked on this amazing project.
No one said stop using atom, it just didn't get enough support.
It was a pleasure accompanying everyone along the way.
PS: and if vscode you can disable the telemetry in a simple click

**Comment by FHU-yezi** [comment]
> Your point is very candid and commendable,. Atom since github was abandoned by microsoft siince they needed to market vscode,
> It is the community of developer out of free time and effort that contribute
> Now atom started coming to life again , gaining popularity especially with 1.60 release,
> they noticed this Since VSCode TRACKS YOU BRO and they felt well... Its about time, They should force everyone to use useless VScode . .... so you get it now

It's sure that the Microsoft version of VS Code is not fully open-sourced, they provided a button to disable trackers, but we don't know it will make sense or not.

But you can build a own VS Code with no Microsoft things, however it will cost lots of time and is it worth?

the core of Atom is open-source, and Election build for "Atom" at first. It is a successful project which makes thousands of Apps based on Election now.

But it has some profermence problem, and needs more RAM to open a big file that other editer such as VS Code and Sublime Text. (Yes the last one is not open-sourced)

**Comment by FHU-yezi** [comment]
Oh I just saw a new editor developed by the Atom group called Zed, hope it can make something amazing after Atom.

It seems to build with Rust, native Application... It will reduce the resource requirements that Election Apps? Maybe, and we can choose one when the public beta version is out.

**Comment by flashtechstudios** [comment]
> VS Code has more powerful features and extensions, let's move on.

Fuck off

**Comment by flashtechstudios** [comment]
> Honestly Its a shame Atom beats VScode 100% till 2morrow. I have both vscode and atom on my localmachine but hardly like HArDLY do I use vscode. Its too bloated, consumes CPU , useless marketplace,etc Atom is not as big but it hits the CORE developer need/ Bill and his evil of buying github and atom which was Defacto shows how evil that man is. He likes to Upset people to accept his useless products. Atom also works better on Linux and Vscode works better on windows. I dont LIKE vscode I want atom and will COntinue to USE ATOM Please Atom Community, Fork Atom and acquire all the resources, The Atom project is too good to be Neglected , Microsoft wants to enforce Vscode and We all SAY NO We all have choices and Editor Preferences , So WHY force anyone...

Blame Satya, Bill left M$.

**Comment by raer0** [comment]
> It's sure that the Microsoft version of VS Code is not fully open-sourced, they provided a button to disable trackers, but we don't know it will make sense or not.

This is incorrect. Microsoft builds the open source VSCode binary, then ships it with a custom product.json with microsoft telemetry and connections to their marketplace. Zed is closed source proprietary software. It is fundamentally not the same as Atom. 

[VSCodium](https://vscodium.com/) is a telemetry free VSCode binary with an open source extension repo. Check it out if you'd like to get started with the closest up-to-date alternative to Atom that you're gonna get.

Perhaps some day, [the atom-community fork](https://github.com/atom-community/atom/) will come together and begin releasing updated/improved new versions of Atom. Until then, if you just need to get back to work soon and need something close, Codium is great.

To anyone here who just wants to express anger about Atom's sunsetting, Microsoft suffocating it, or whatever else, all I have to say is: Go somewhere else.

**Comment by LittleTasteOfHeaven** [comment]
@flashtechstudios Use vscode and be happy with it

**Comment by kaosine** [comment]
> Perhaps some day, [the atom-community fork](https://github.com/atom-community/atom/) will come together and begin releasing updated/improved new versions of Atom. Until then, if you just need to get back to work soon and need something close, Codium is great.


I'd recommend checking out the discussions over there and joining in on the discord. Which is now linked in the readme since I got that update approved earlier. Not to mention they have builds published on azure linked in there as well. I'm also attempting to archive and get a good starting point for documentation going forward in my own repos. (the [ac-website one](https://github.com/kaosine/ac-website/) which may eventually become official once I get it in working order. That too is open for any suggestions. I know right now quite a few people over there in the discussions and on discord have been trying to rebuild and migrate the apm repository code from scratch and update the apis it relies on as well. I think one person there has electron pushed to 13 and is working on getting it more current as we speak.

**Comment by raer0** [comment]
> > Perhaps some day, [the atom-community fork](https://github.com/atom-community/atom/) will come together and begin releasing updated/improved new versions of Atom. Until then, if you just need to get back to work soon and need something close, Codium is great.
> 
> I'd recommend checking out the discussions over there and joining in on the discord. Which is now linked in the readme since I got that update approved earlier. Not to mention they have builds published on azure linked in there as well. I'm also attempting to archive and get a good starting point for documentation going forward in my own repos. (the [ac-website one](https://github.com/kaosine/ac-website/) which may eventually become official once I get it in working order. That too is open for any suggestions. I know right now quite a few people over there in the discussions and on discord have been trying to rebuild and migrate the apm repository code from scratch and update the apis it relies on as well. I think one person there has electron pushed to 13 and is working on getting it more current as we speak.

I'm already there.

**Comment by muhammadanas365** [comment]
I can't say much, but I have to say, using **Atom** isn't the easiest thing to do. VS Code is so much more intuitive and beginner friendly. Sorry Atom lovers, but it's time we move on to VS code!

**Comment by kaosine** [comment]
Lmao no *cough* AtomCommunity/Atom *cough* especially since we're going to rebrand and fix things better than atom did for the past few years since they were diverted elsewhere by M$

**Comment by muhammadanas365** [comment]
Haha yes!

**Comment by stech11845** [comment]
> > VS Code has more powerful features and extensions, let's move on.
> 
> Move on your own with VScode, Bill gates deserves to be dragged to court. We all have choices and Preferences. I dont WanT vscode

Bill Gates has left, Satya

**Comment by iamunadike** [comment]
> > > VS Code has more powerful features and extensions, let's move on.
> > 
> > 
> > Move on your own with VScode, Bill gates deserves to be dragged to court. We all have choices and Preferences. I dont WanT vscode
> 
> Bill Gates has left, Satya

If you really do a comparison of vscode to atom, you will find out that atom esp at 1.60 is very good
in terms of extensions,  for some languages like you have greater support , vscode has a marketplace of extensions no doubt , but most extensions do simple tasks as tracking your code, providing snippets and autocomplete. these extensions themselves can also slow down vscode if too much
Atom is more hackable than vscode, the layout  is better, I still have sound autocomplete and good extensions enough to put me on a go
Atom may be in an unsupported state but in no way is it less of a good developer tool in 2022

**Comment by muhammadanas365** [comment]
Whatever Atom may contain, VS code is just so much better!

**Comment by iamunadike** [comment]
> Whatever Atom may contain, VS code is just so much better!

Nice but give me top 15 stats , Where VScode is so much better than Atom?
I am listening....

**Comment by muhammadanas365** [comment]
1. It has Github support, which is seamless and easy to navigate, and Direct access to Github Codespaces, Github Copilot, and all Github Features
2. It supports thousands of themes (extensions)
3. Not to forget the built-in docker and Heroku support
4. Built-in terminal which is awesome
5. Syncing is possible, so you can always sync your VS code account to another computer, never losing the settings you set previously
6. So many IntelliSense features, for almost every language out there
7. Not to forget the built-in support for Jupyter Notebooks
8. A very friendly UI and easy to navigate through
9. There is this extension I use to practice Leetcode questions without leaving VS Code. Reading the problem, coding the problem, and submitting the code
10. it has extensions to help you view excel, CSV, pictures, and music files from your codebase, without ever leaving it.
11. Very frequent updates, like sometimes thrice a week, or at least once a week, and one major update in a month
12. Built-in browser feature, so you can render your HTML directly in VS code
13. Live Share so multiple people (teams) can code in together in one codebase
14. Live Server, which automatically refreshes your browser when dealing with HTML (Web  Dev stuff)
15. Auto-save feature!

Now you tell me why Atom is anyway better than VS code?

**Comment by iamunadike** [comment]
I said "so much better" than atom
Of all you said above , mostly integrations were your highlists:
- Live Server 
- GitHub all the features > this is really reflected to be true because Microsoft owns both Github and Vscode so .....
- Extensions , Presently Vscode has more support and is backed by microsoft so definitely yes


If you used vscode, it depends on what you want and some stuff you said above is already in atom too
Github personal tool is vscode , used to be atom before microsoft bought it
Do a side by side research on atom and find out was it is called the most hackable editor

**Comment by muhammadanas365** [comment]
I know, Atom is called the **most hackable editor** but it still doesn't make it any better than Atom. Github owns ATOM, and Github is owned by Microsoft. So it could be made much more like VS code but didn't. 

Of course, I'm biased!

**Comment by iamunadike** [comment]
Then again imagine if every project just dies simply because some new product started to make temporary waves,
U probably might not about Sublime, Vim, Emacs, 
Funny enough some hackers still prefer vim and emacs to vscode
Atom is more opensource too than vscode , since vscode tracks you with telemetry 
Atom has a integration with github and gitlab , though that is not working well as expected now
Atom has a better Navigation UI than vs code 
Atom may not need a terminal if you using linux , but definitely you can do have git commit supports and branching etc , without codespaces since it is a pure microsoft product and ofcourse microsoft does not share... until it bites them hard
Atom intelllisense is still very much good 
Atom has docker support and you docker is quite old since it is good but has a slow maintenance , so to make up cloud host companies have a way of giving you virtual contaiiners , but definitely is still good
Atom has a smaller community, Microsoft initiated that since it felt like github -> atom, but since microsoft bought github. It is more of github -< vscode 
Atom supports any language, the core of atom is so good that is starts up gracefully with power and cpu stressing issues after how many years,  vscode started to catchup in this areas recently
Atom does not have it all at the moment but as it continues to gain more community recognition , it will come back bigger

**Comment by muhammadanas365** [comment]
Agreed! But I even prefer Sublime over Atom. Sublime and VS Code are my personal favourite

**Comment by iamunadike** [comment]
Every editor has their pros and cons

**Comment by muhammadanas365** [comment]
Agreed!
